Formula/gcc@10.rb

```diff
@@ -114,6 +133,81 @@ def add_suffix(file, suffix)
     File.rename file, "#{dir}/#{base}-#{suffix}#{ext}"
   end
 
+  def post_install
+    on_linux do
+      gcc = bin/"gcc-#{version_suffix}"
```

Comment:
I think all of these upstream PRs are missing the method definition for `version_suffix`.  We definite it as a local variable inside `install`, but not in the `postinstall` block.  Rather than define it twice, I guess we defined a method to return it.

Reply:
Fixed. Nice catch.

Reply:
Yeah it took me way too long to figure out this was missing when I was trying to get the `gcc@10` PR to work in linuxbrew-core.  So now it's etched into my memory! 😂